src/modules/admin/components/AdminDashboard/AdminDashboard.tsx

```diff
@@ -69,7 +69,7 @@ const navigationItems = (colony: ColonyType): NavigationItem[] => [
   {
     id: 1,
     title: MSG.tabProfile,
-    content: <Profile colony={colony} />,
+    content: <ProfileEdit colony={colony} />,
```

Comment:
`ProfileEdit` is still missing the heading & underline bit.

Here's how it is now:

<img width="1454" alt="Screen Shot 2019-09-27 at 11 09 40 AM" src="https://user-images.githubusercontent.com/3052635/65784286-55b60380-e117-11e9-95df-500ebbdf54e1.png">


And here's the figma:

![Profile](https://user-images.githubusercontent.com/3052635/65784242-3fa84300-e117-11e9-809f-06072680b037.png)